Basics of reading ASDF files.ipynb

```diff
@@ -0,0 +1,1084 @@
qVtINJOCdEWX3KqIi2p0EBChd1d06Ftse0bH8ZbrKhYlUtWM+EBVa7zh++dJPenPrFZJp+nNp0NVOmekAEyI+SkahQ8gLs6AdAJ5oFrj5tYOQzwFZYqGFEQlNAAym7LCQDO02kEHx7PQGbEfzeWIOSQhJV1SbLFmQk+OOhC3yw7pXhJRmHrFw620SAct2iqBmYjnOqFYWSsj2iFfU7Lgw9QOAsKJkrlIY4dsN5HPmafmZzznDuzXfXzuzssB0HnmU272tjZwRytkDxbcLJZbJP0W1gS485wqQxLoPAPghiKmbYGPvuyX8Oy3/TjggfpwC1Pb6OhrJN035IIiVB1drJ+TDnb1FRXShwousnOD+lCLZJpQs7PB3CLTdq8337aoDlG/oxQqNU083T8MGTpW4t7nvRUA8KiPfheqh0ZEp3K+Jz2N+yJwsEyojSnOZLH7LvAld8JPD/zwS/4Mv/wXL4Yft7ATdvQlcxv1FrSRk6b0Sw5wTByHN3BjXquQBCD3QGuRLDXwuzlsaVBsUuBdrTN1O6QBxWaC6hChmHzXxpOt8XymWA4s1L+4onQo9BmDOO1Td3H7034fN7z55vhMtIOAa17/cQDAAz/zTBn8ZCgYiltxaoFjFYIH7NkeXM8j5IFUWMFASrdMrVayICXqBw7wBrAqEAOQBGQXM7RDj6c/6W78/pUfxJXvfTXQWKIreykLkFe46Cga4MWN+b5v+0VcnY1w9dtfR9pTKDy4DgFyIx/zt9ohkaJ6xeMbnv1J/OqxTwIAnvLpl2Hr/tUYw+F6jEvon+lylPQgpXldilS2I6GeaxNLpG1t4JYd7IwoXlJ3KEszIorhRewc9wQxaDRLRMaImjIRvB1IlthKiLZ9optcHxGA6rDj56JapImJJhE1+PirFkju7UOrVfTw1yx7uBHvSRMMTNW9n2zfojriYCoaRPj6wSEuFY2nbN6u4EDULIWoS6zWfERqM9FjpVOpTKmJSscDcB7wvd/417i+dxb/j9//LqTTzpUWrHzu8v6MPOduINcqlT1WstqSUmjAEffX3jbvnYNsgusTddLMt2Dp+GuHfF9aYMyyYkG5XbdHah2Sdv5pvpIe7LMpUK+S7dD08bYPJA33DnXXcaBmoLHuKbrvkrrsKL52wD/XDtveZsD9b3hEi8PXwqNe/m9RC0qTTbssmehEkCBFlv0x0yldQLKZuLjammGMmp1Uj01smO5fDBEtagf8GWqFT2dBIGUTh7N0EaJYXZ1yenJoRiICTxGV/EAn6G6HnJy1WkVdJdG5ZxSCZ66T5qj43HSbjwowZThQh1fbF3fbUPQGYsuMok1xoGg1SgwRBR0F9RInbX1telOqPkFF2gpnM6mXg0gqPH4w/FkxwkCsqjGfSXh5L3qNYrdDmRQpBBDhfRXUK7qlpyZFFjT0zLZyYh4fOFkHOjgmV/lIz/mUm3kYOKS9Fu1+DmQedpLC9x16ZzMEy9OcW3JI9tIIB8MwnyeZWaYFVzZy+CGRE2nG31uco5h2flWNZC8ltH5pBbuZA0crZHmLcr9Asi1iYQ/gqhnai31mFNlOE9AOPdKJjfSGimk12yYUHvlqibZOkJ3sQfOpTADmlzpuvHlAur4AvjwkatBnUGS+Q0pgfjk3ZF9IhchmHyH1tN3LPa76h3bAU75a8XUwM87IRsQgT96X3DCMB9x6A2MDzE4etVOkb3gPZDO6r9IDtIE7UiEsUgq7r5zAmADnLNLUY/7gEkLPCZ1nYFdq9AcVFvMCfjunPmPQwu5lCKs1kswDD/bjYOnzgJB3GzQORAVohhBst6Gmc6G8CiBcuoA91Uc78BicmOJnHv2n+Mk//05GRcjznFT8Ger6VDfmwYNSs+yBpRZhkSBbLeEaEVnvZ1g6PsHnn/oHMVMpnZsoCWiXWw6DtUXvTIbqkOMgO+ZgUWxZCset5FMFDoY6YCV7zIGCJTKqAud2wGGtWFugXmR8vZMMcEQPIaifLUVMnUm2mA6Rj6KU4fN1iW/6sx/m75paOjJXmV+mpdHzKxuY2sbDSjPiYJXMLDsOW4tgAvKdBPUGEcxvfPJn8O5PPgHJNOFwu0dUz/WDUNS2YyIEQWqXgiDhnfDfCJLMIGOl2Ttqh5rWTtLQDkLMsDJOYhSEMq/XmMSeLAwWxx2KrUR660J8DrJtOWRlIX5/K4Gg2mqgZbvtgP17yVRoztLI8I4IBqirrlkiLbi4hFSthjEnNU0VXc2VZqtplpLU58hQwsOrjzEVrIvhZ9Y7n8TGC5W8JOKs69Zag4993y/iK97043A5D8dAd8C1dXdYjve+aF6zfb5G3Zc08iM5kDOo+ial1lxf/k6Fh+Uaaql7vsfw1GShYECIMoZsos8Dh7BGhibKZIKwHQH3/sYjnKqzreTtSAjk4pDB3nW86IsjFDzrJpOWIrBspUtLHG0+VTidtF02Z4ZTbytEOJAhdnyY+D1EmfSG1RyJIBUCJvDn8Ibk78ymHRwdE8dniBNzOut+FqtH+M9KaX3hh26N0zyTvk0s6s33AtJZl1llXGBEgdCWacmhqTzk4W6cUVukUKdh/lK90lFpAH+vdpHpje01QFSsp73NEKMOFEpXTr/YYYCgDljWkdKr1hGhT+M714iiCdlUwuqeOIF57H4cyNIFDtjbO81XohC/MwhFwH/67reKbocn/pAE1GPEaAAVhTcjACs1kZGKm3U6tUg3Mw5NJgCNpCLXFOo6sdnbKSF61b4BgF1Iv50MRjosplOLZG5hSoviLPne+YmW+UVzg3rNwaYBw6v3cOXRTdz5Fb+DSy7Zhi88mrGjqWGrz14usZHbGkgkiM840gaxRmFAfVC+azE4laI9O0DYz9GMA8pLa5SHPIMsPU/JycSi3erHtO1kTspSHV9GNC/50Tnuevbb8JKnfwqmldDNYZBqHk8HnJxK833JQapUUBqiVse0st54wK+0jA5YJAjiRHMrLe58za2472VviEOgilzrQ06CBQOwn8XY
```

Comment:
Hmm... this is a good use case for a "stable" version on the docs, so that documentation links in notebooks will stay current with the latest release of asdf.  I'll go ahead and set up that version now, which mean the link target can change to this:

https://asdf.readthedocs.io/en/stable/api/asdf.info.html#asdf.info

(2.7.1 --> stable)